misaligned.js

```diff
@@ -1,16 +1,43 @@
 const {expect} = require('chai')
+const {ActualColorCodePairs} = require('./actualColorcodePair')
+var colorCodePairList = [];
+
+var majorColors = ["White ", "Red ", "Black ", "Yellow", "Violet"];
+var minorColors = ["Blue ", "Orange", "Green ", "Brown ", "Slate "];
 
 function print_color_map() {
-    const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
-    const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
-    for (let i = 0; i < majorColors.length; i++) {
-        for (let j = 0; j < minorColors.length; j++) {
-            console.log(`${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}`);
-        }
+  for (let i = 0; i < majorColors.length; i++) {
+     for (let j=0; j < minorColors.length; j++) { 
+         colorCodePairList.push(`${i* 5 + j} | ${majorColors[i]} | ${minorColors[j]}`); }

```

Comment:
The number 5 would lead to confusion or error in future when someone else is maintaining the code.
Also, will this code work if major color or minor color length is other than 5. Please think over it and write test cases accordingly to cover such cases.